lib/public/services/detectors/detectorsProvider.js

```diff
@@ -11,34 +11,56 @@
  * or submit itself to any jurisdiction.
  */
 
-import { Observable } from '/js/src/index.js';
+import { Observable, RemoteData } from '/js/src/index.js';
 import { getRemoteData } from '../../utilities/fetch/getRemoteData.js';
 
 /**
- * Service class to fetch detectors from the backend
+ * Service class to fetch detectors
  */
 export class DetectorsProvider extends Observable {
     /**
      * Constructor
+     * @param {DetectorsProvider} parentInstance paranet instance
+     * through which child-instances can share successfuly fetched detectors data therby reducing number of requests.
      */
-    constructor() {
+    constructor(parentInstance = null) {
```

Comment:
WTH is parent instance?

Reply:
that H that is written above

Reply:
https://github.com/AliceO2Group/Bookkeeping/pull/1288#discussion_r1424232887